warehouse/templates/accounts/unrecognized-device.html

```diff
@@ -0,0 +1,17 @@
+{# SPDX-License-Identifier: Apache-2.0 -#}
+{% extends "base.html" %}
+{% block title %}Unrecognized device{% endblock %}
+{% block content %}
+  <div class="horizontal-section">
+    <div class="site-container">
+      <h1 class="page-title">Unrecognized device</h1>
+      <p>We did not recognize this device. Please check your email for a login confirmation link.</p>
+      <p>
+        You should have received an email from <strong>noreply@pypi.org</strong> with the subject line "<strong>Unrecognized login to your PyPI account</strong>".
+      </p>
+      <p>
+        If you did not make this change, you can email <a href="mailto:admin@pypi.org">admin@pypi.org</a> to communicate with the PyPI administrators.
```

Comment:
"If you did not make this change"? Seems wrong?

Reply:
821531c